boards/DCM/Src/App/states/App_SharedStates.c

```diff
@@ -0,0 +1,36 @@
+#include "states/App_SharedStates.h"
+
+void App_SharedStatesRunOnTick1Hz(struct StateMachine *const state_machine)
+{
+    struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct RgbLedSequence *rgb_led_sequence =
+        App_DcmWorld_GetRgbLedSequence(world);
+
+    App_SharedRgbLedSequence_Tick(rgb_led_sequence);
+}
+
+void App_SharedStatesRunOnTick100Hz(struct StateMachine *const state_machine)
+{
+    struct DcmWorld *world = App_SharedStateMachine_GetWorld(state_machine);
+    struct DcmCanTxInterface *can_tx      = App_DcmWorld_GetCanTx(world);
+    struct DcmCanRxInterface *can_rx      = App_DcmWorld_GetCanRx(world);
+    struct BrakeLight *       brake_light = App_DcmWorld_GetBrakeLight(world);
+    struct InverterSwitches * inverter_switches =
+        App_DcmWorld_GetInverterSwitches(world);
+
+    // Only need single torque command since same command sent to both inverters
+    float torque_command =
+        App_CanMsgs_dcm_invl_command_message_torque_command_invl_decode(
+            App_CanTx_GetPeriodicSignal_TORQUE_COMMAND_INVL(can_tx));
+
+    bool is_brake_actuated =
+        App_CanRx_FSM_BRAKE_GetSignal_BRAKE_IS_ACTUATED(can_rx);
```

Comment:
can we pass this into set light status directly in without this bool?

Reply:
thought it'd be cleaner than having a long named function as an argument

Reply:
done